homeassistant/components/sensor/const.py

```diff
@@ -248,7 +248,7 @@ class SensorDeviceClass(StrEnum):
     """
 
     PH = "ph"
-    """Potential hidrogen (acidity/alkalinity).
+    """Potential hydrogen (acidity/alkalinity).
```

Comment:
The same typo is in number/const.py.

Reply:
Thanks! Just noticed that on the docs, corrected here as well 👍